src/promnesia/__main__.py

```diff
@@ -245,15 +276,46 @@ def cli_doctor_server(args: argparse.Namespace) -> None:
     logger.info('You should see the database path and version above!')
 
 
+def _ordinal_or_name(s: str) -> Union[str, int]:
+    try:
+        s = int(s)  # type: ignore
+    except ValueError:
+        pass
+    return s
+
+
 def main() -> None:
     # TODO longer, literate description?
 
+    def add_index_args(parser: argparse.ArgumentParser, default_config_path: PathIsh) -> None:
+        """
+        :param default_config_path:
+            if not given, all :func:`demo_sources()` are run
+        """
+        parser.add_argument('--config', type=Path, default=default_config_path, help='Config path')
+        parser.add_argument('--dry', action='store_true', help="Dry run, won't touch the database, only print the results out")
+        parser.add_argument(
+            '--sources',
+            required=False,
+            action="extend",
```

Comment:
Converting to `append`, you lose this syntax: `--sources viber telegram`,
and forced to use `--source viber --source telegram`
(notice that then the options cannot be in plural).

Will add SO workaround in `compat.py`.

Reply:
Just pushed and waiting for circleCI's opinion.

Locally, my tox fails with ` ModuleNotFoundError: No module named 'dal_helper'`.
What should id do?

Reply:
OK, done!
(in a separate commit, to isolate the compatibility issue).

Reply:
> ModuleNotFoundError: No module named 'dal_helper'
I think it would also besolved by `git submodule update --init --recursive`

Hmm. So all tests passing locally for me! But now github fails in weird ways.. I'll try to investigate why